parsec/backend/cli/run.py

```diff
@@ -449,6 +449,9 @@ async def _serve_backend_with_asgi(
         {
             "bind": [f"{host}:{port}"],
             "accesslog": logging.getLogger("hypercorn.access"),
+            # Timestamp is added by the log processor configured in `parsec.logging`, here
+            # we configure req line + rep status + rep body size (e.g. "GET /foo 1.1 404 823")
+            "access_log_format": "%(r)s %(s)s %(b)s",
```

Comment:
```suggestion
            # we configure req line + rep status + rep body size (e.g. "127.0.0.1 GET /foo 1.1 404 823")
            "access_log_format": "%(h)s %(r)s %(s)s %(b)s",
```
It would be nice to include the IP of the client

Reply:
Updated, I've also added the request time